homeassistant/components/utility_meter/sensor.py

```diff
@@ -260,7 +261,8 @@ async def async_added_to_hass(self):
                 minute=self._period_offset.seconds % 3600 // 60,
                 second=self._period_offset.seconds % 3600 % 60,
             )
-
+        elif self._period == NONE:
```

Comment:
This is not necessary (it does nothing!)

Reply:
That's right, but if I omit this the code does not pass, because the const NONE is never used.
When NONE is selected, there is no need to create an `async_track_time_change`, but not using this value at all is a no-pass

Reply:
don't worry, just delete this as it is unnecessary